nearlib/wallet-access-key.js

```diff
@@ -0,0 +1,128 @@
+/**
+ * Wallet based account and signer that uses external wallet through the iframe to signs transactions.
```

Comment:
Is the iframe comment still correct?

Reply:
Oh. Didn't update that top level comment.